[invalid.py]
  [test_params_locator_value_invalid_value[innerText-\]]
    bug: 1869538
    expected: FAIL
